src/replace.ts

```diff
@@ -9,7 +9,7 @@ export function replace (source: string, selector: string, stringTransformer: TS
     const reversedReplacements = replacements.reverse();
     let result = source;
     reversedReplacements.forEach((replacement, index) => {
-        if (replacement) {
+        if (replacement !== null && replacement !== undefined) {
```

Comment:
Sorry to be so picky on such a small change, but If you don't mind I prefer `!= null` for `null`/`undefined` checks!

Reply:
Be picky 😄 The size of a PR does not matter

I amended my commit with the requested code change. 
Although, I find it less obvious but I don't really care about that. I'm fine with the requested change🙂